### PR TITLE
More explicit instructions & expectations

### PR DESCRIPTION
**Pull request by kevintc**
Following internal discussion about our senior test, here are a few updates to make the Readme more explicit.
For feedbacks - reach out in Slack.

### CODE REVIEW

**Review by MarcFRICOU — COMMENTED** [review]
We never mention any code structure, like hexagonal and DDD. Is that an issue? One of the reasons why we rejected engineers is the code structure.

Also: Shouldn't we explicitly indicate that we expect the CLI to consume the API?

File: README.md [hunk 1]
```diff
@@ -47,18 +51,27 @@ Topical Calcipotriene Treatment for Breast Cancer Immunoprevention, France
...
 We expect you to spend no more than 5 hours on this assignment.
 
+This assignment is limited in scope and could be solved by writing all the code in a single file. Still, we want you to architect your tests and your code as if you were building the first parts of a larger-scale software. Imagine that a lot of features are going to be added in the future, by other engineers. Focus on maintainability and extensibility, even though it is clearly over-engineering at this point.
+
 You are allowed to use the technologies of your choice, but if you are looking for inspiration use [ours](https://stackshare.io/inato/marketplace). You are encouraged to make good use of open-source code.
 
+## Expectations
+- [ ] You followed the instructions 
+- [ ] We can install & query the Web API
+- [ ] We can install & query the CLI
```
Comment (bastoche) [inline README.md:71]:
```suggestion
- [ ] We can run the CLI
```

File: README.md [hunk 2]
```diff
@@ -47,18 +51,27 @@ Topical Calcipotriene Treatment for Breast Cancer Immunoprevention, France
 # Instructions
 
 - [ ] Clone this repository (do **not** fork it)
-- [ ] Implement the features step-by-step, documenting your architecture and design choices along the way
+- [ ] Implement the features step-by-step (your commit history should be clear to follow)
+- [ ] Document your architecture and design choices along the way
+- [ ] Provide documentation on how we should use your code
 - [ ] Publish it on GitHub (or equivalent)
-- [ ] Send us the link and tell us approximatively how much time you spent on this assignment
+- [ ] Send us the link and tell us approximately how much time you spent on this assignment
```
Comment (bastoche) [inline README.md:58]: 👍

File: README.md [hunk 3]
```diff
@@ -47,18 +51,27 @@ Topical Calcipotriene Treatment for Breast Cancer Immunoprevention, France
 # Instructions
 
 - [ ] Clone this repository (do **not** fork it)
-- [ ] Implement the features step-by-step, documenting your architecture and design choices along the way
+- [ ] Implement the features step-by-step (your commit history should be clear to follow)
+- [ ] Document your architecture and design choices along the way
+- [ ] Provide documentation on how we should use your code
```
Comment (bastoche) [inline README.md:56]:
```suggestion
- [ ] Provide instructions on how we can run your code
```

File: README.md [hunk 4]
```diff
@@ -47,18 +51,27 @@ Topical Calcipotriene Treatment for Breast Cancer Immunoprevention, France
...
-
 We expect you to spend no more than 5 hours on this assignment.
 
+This assignment is limited in scope and could be solved by writing all the code in a single file. Still, we want you to architect your tests and your code as if you were building the first parts of a larger-scale software. Imagine that a lot of features are going to be added in the future, by other engineers. Focus on maintainability and extensibility, even though it is clearly over-engineering at this point.
+
 You are allowed to use the technologies of your choice, but if you are looking for inspiration use [ours](https://stackshare.io/inato/marketplace). You are encouraged to make good use of open-source code.
 
+## Expectations
+- [ ] You followed the instructions 
+- [ ] We can install & query the Web API
```
Comment (bastoche) [inline README.md:70]:
```suggestion
- [ ] We can run & query the Web API
```

**Review by bastoche — APPROVED** [review]

**Pull request merged**